package.json

```diff
@@ -5,7 +5,7 @@
   "workspaces": {
     "packages": [
       "packages/*",
-      "apps/*",
+      "apps/website",
```

Comment:
This will break the demo apps as they rely on the local built clarity packages for dependencies. This will prevent local installs from working which we use for day to day testing/dev on core. Let's hold off on changing this line. There are a couple of related PRs and work that I am going through that this would effect.

Reply:
@gnomeontherun I think had some experience testing out the yarn zero install, he might be able to add some details.

Reply:
I tested the `zero-install` for yarn - it's not working that smooth as they promote it - also like I mentioned in the description there are issues with linking directly by the path and also keeping the `.yarn/cache` inside the repository will add a huge drawback on checkouts and overall git speed. 

Also, there some issues with the yarn 1x & 2x that are making them incompatible or at least with this state of linking packages between sub-projects. It was super hard to test it. 

I think that `yarn zero-install` is something that we could investigate next year after they release the stable version and provide better support.

Reply:
We can make this change to exclude the apps from the yarn workspace but to keep the apps working for local installs in each app update the `@clr/core` package path (excluding website)

from `"@clr/core": "./packages/core/dist/core"`
to `"@clr/core": "../../dist/core"`

This will allow each app to be able to run yarn independently and still install. When they were part of the workspace the install only worked if the path started at the root of the workspace.

Reply:
Gonna update the `package.json` files to address the paths and run a few tests to see if all the packages are building correctly. 

Also will create a subtask to install the dependencies for each of the sub-projects.

Reply:
Make sure this aligns with @bbogdanov's work please.

Reply:
@bbogdanov showed me a really good solution for our local package usage that should clean up a lot of our build code. I think this change will still work fine for his update but +1 to double check.

Reply:
Yap, we had a chat with @bbogdanov and sync our approach on the issue. Manage to mark some point where we need to sync. 

So by mentioning it, we decide the one that manages to merge his PR last will have to solve the potential `git-conflict` inside the version inside `apps/`. (I'm not changing it right now - cause I'm not sure what will be the final version of the package that we gonna put (`@latest` or `=4.0.6` "exact version") - and wanted to keep this PR working at the moment)

And by saying that .. I hope to merge mine before him! 😆 

Update `apps/` packages to have their own `yarn.lock` file and be stand-alone apps. I had the idea to have `pre-hook` for installing `yarn` packages but notice that `yarn install` will be called without the need of doing it on every `build` call and even that the apps are pretty small there is no need for that (also could result of creating different `yarn.lock` file that had to be commit/track).

Reply:
Now that this has rebased onto `next` I think this particular line will still break the demo apps. It excludes them from the workspace so when yarn is ran at the top level of the repo or in the specific demo app directory it will fail.

Reply:
@coryrylan update to address the changed path of the `core` dist

Reply:
The path update looks good to me, I think the cds rename is all thats left